src/basics.md

```diff
@@ -1242,6 +1243,50 @@ fn main() {
 }
 ```
 
+[ex-datetime-arithmetic]: #ex-datetime-arithmetic
+<a name="ex-datetime-arithmetic"></a>
+## Add or subtract dates and times using checked arithmetic
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Calculates and displays the date and time two weeks from now using
+[`DateTime::checked_add_signed`] and the date of yesterday using
+[`DateTime::checked_sub_signed`]. The methods return None if the date and time
+cannot be calculated.
+
+Escape sequences that are available for the
```

Comment:
[Display formatted date and time](https://rust-lang-nursery.github.io/rust-cookbook/basics.html#display-formatted-date-and-time) is already described in the cookbook. if you want to have one line explaining this, I would forward the reader to the relevant recipe.

Reply:
If I understood correctly what you meant, I feel like this would indeed reduce redundancy, but it would also disorient the reader. Here, it cannot be assumed that the reader is familiar with all the recipes. That's why I would probably prefer to just repeat something as simple as this, rather than link to another recipe.

Reply:
> If I understood correctly what you meant

Sorry for not being clear. You guessed it right :-)

My thoughts behind forwarding the reader: it makes the cookbook more discoverable (help the reader to be familiar with several recipes) and each recipe focuses solely one item (less redundancy).

I let you discuss it with @budziq.